documentation/asciidoc/topics/yaml/values.yaml

```diff
@@ -0,0 +1,102 @@
+#Build configuration
+images:
+  server: quay.io/infinispan/server:13.0
+  initContainer: registry.access.redhat.com/ubi8-micro
+#Deployment configuration
+deploy:
+  replicas: 1
+
+  container:
+    extraJvmOpts: ""
+    storage:
+      size: 1Gi
+      storageClassName: ""
+      ephemeral: false
+    resources:
+      limits:
+        cpu: 500m
+        memory: 512Mi
+      requests:
+        cpu: 500m
+        memory: 512Mi
+
+  security:
+    secretName: ""
+    batch: ""
+
+  expose:
+    type: Route
+    nodePort: 0
+    host: ""
+    annotations: []
+
+  logging:
+    categories:
+      - category: com.arjuna
+        level: warn
+      # No need to warn about not being able to TLS/SSL handshake
+      - category: io.netty.handler.ssl.ApplicationProtocolNegotiationHandler
+        level: error
+
+  makeDataDirWritable: false
+
+  nameOverride: ""
+
+  resourceLabels: []
+
+  infinispan:
+    cacheContainer:
+      name: default
+      security:
+        authorization: null
+      transport:
+        cluster: ${infinispan.cluster.name:cluster}
+        node-name: ${infinispan.node.name:}
+        stack: kubernetes
+    server:
+      endpoints:
+      - securityRealm: default
+        socketBinding: default
+      - connectors:
+          rest:
+            restConnector:
+              authentication:
+                mechanisms: BASIC
+        securityRealm: metrics
+        socketBinding: metrics
+      interfaces:
+      - inetAddress:
+          value: ${infinispan.bind.address:127.0.0.1}
+        name: public
+      security:
+        credentialStores:
+        - clearTextCredential:
+            clearText: secret
+          name: credentials
+          path: credentials.pfx
+        securityRealms:
+        - name: default
+          propertiesRealm:
+            groupProperties:
+              path: groups.properties
+            groupsAttribute: Roles
+            userProperties:
+              path: users.properties
+        - name: metrics
+          propertiesRealm:
+            groupProperties:
+              path: metrics-groups.properties
+              relativeTo: infinispan.server.config.path
+            groupsAttribute: Roles
+            userProperties:
+              path: metrics-users.properties
+              plainText: true
+              relativeTo: infinispan.server.config.path
+      socketBindings:
+        defaultInterface: public
+        portOffset: ${infinispan.socket.binding.port-offset:0}
+        socketBinding:
+        - name: default
+          port: 11222
+        - name: metrics
+          port: 11223
```

Comment:
I'd like to revert this back and keep it minimal for the procedure. We also need the attributes for the server images and schema versions.

Reply:
yes, this was an overkill